fetch/origin/no-cors.window.js

```diff
@@ -76,3 +76,65 @@ promise_test(async function () {
   assert_equals(json[0], "null");
   assert_equals(json[1], "null");
 }, "Origin header and POST navigation");
+
+function referrerPolicyFucnctor(referrerPolicy, isCors, expectedOrigin) {
```

Comment:
Instead of "isCors", let's make this "crossOrigin" or "isCrossOrigin" or some such as it doesn't really relate to CORS.